src/main/java/frc/robot/subsystems/GBSubsystem.java

```diff
@@ -23,10 +23,10 @@ public String getLogPath() {
 
 	@Override
 	public final void periodic() {
-		Logger.recordOutput(getLogPath() + "Current Command", getCurrentCommandName());
+		Logger.recordOutput(getLogPath() + "CurrentCommand", getCurrentCommandName());
 		subsystemPeriodic();
 	}
 
-	protected abstract void subsystemPeriodic();
+	protected void subsystemPeriodic() {}
```

Comment:
Why?

Reply:
i dont want to force people to use it, and i need this so people override it instead of the regular periodic. its kind of forcing to use super in function